cmd/ddev/cmd/scripts/test_ddev.sh

```diff
@@ -62,8 +65,9 @@ ddev add-on list --installed
 header "mutagen situation"
 
 echo "looking for #ddev-generated in mutagen.yml in project ${PWD}"
+echo
 if [ -f .ddev/mutagen/mutagen.yml ]; then
-  if grep '#ddev-generated' .ddev/mutagen/mutagen.yml; then
+  if grep -q '#ddev-generated' .ddev/mutagen/mutagen.yml; then
```

Comment:
Excellent. I always do `>/dev/null` and `-q` is better. And regardless, it's good to lose that output here.